public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -867,7 +832,7 @@ describe('RuleList', () => {
 
         renderRuleList();
 
-        await waitFor(() => expect(mocks.api.fetchRules).toHaveBeenCalledTimes(1));
+        await waitFor(() => expect(mocks.api.fetchRules).toHaveBeenCalled());
```

Comment:
We're now testing with an arbitrary number of datasources, so `fetchRules` was being called more times (and could change in the future).
This assertion was mainly making sure that the UI had "settled" a bit before checking for the button's presence, so I switched it to a `toHaveBeenCalled`